src/test/groovy/BusinessScenariosTests.groovy

```diff
@@ -0,0 +1,162 @@
+import com.getbase.Client
+import com.getbase.Configuration
+import com.getbase.models.Contact
+import com.getbase.models.Deal
+import com.getbase.models.Stage
+import com.getbase.models.User
+import com.getbase.services.ContactsService
+import com.getbase.services.DealsService
+import com.getbase.services.StagesService
+import groovy.util.logging.Slf4j
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.format.DateTimeFormatter
+import spock.lang.Requires
+import spock.lang.Shared
+import spock.lang.Specification
+
+import static org.awaitility.Awaitility.await
+
+/**
+ * Created by przemek on 06.10.2016.
+ */
+
+@Slf4j
+@Requires({ System.getProperty("integrationTests").equals("true") })
+class BusinessScenariosTests extends Specification {
+
+    @Shared
+    def client
+
+    @Shared
+    User userSalesRep, userAccManager
+
+    @Shared
+    Contact createdContact
+
+    @Shared
+    String personContactName = 'person assigned to a manager'
+
+    def setupSpec() {
+        client = new Client(new Configuration.Builder()
+                .accessToken(System.getProperty("BASECRM_ACCESS_TOKEN"))
+                .build())
+
+        userSalesRep = client
+                .users()
+                .get(convertUserId(System.getProperty("userSalesRep")))
+
+        userAccManager = client
+                .users()
+                .get(convertUserId(System.getProperty("userAccManager")))
+
+        def testContactName = 'Foo Contact'
+
+        assert client
+                .contacts()
+                .list(new ContactsService.SearchCriteria().name(testContactName))
+                .size() == 0
+
+        createdContact = client.contacts().create(new Contact(name: testContactName, ownerId: userSalesRep.id, isOrganization: true))
+        await().until {
```

Comment:
it's not necessary -> you can assume that "createdContact" is created after call to client's "create"